MovieApp/ViewControllers/MovieListViewController.swift

```diff
@@ -0,0 +1,104 @@
+//
+//  Created by Nikola Đokić on 12.04.2022..
+//
+
+import Foundation
+import UIKit
+import SnapKit
+
+class MovieListViewController: UIViewController{
+    
+    private var searchBarView: SearchBarView!
+    private var homeScreenMoviesView: HomeScreenMoviesView!
+    private var focusedHomeScreenView: HomeScreenFocusedView!
+    private var navigationBarAppName: UIView!
+    private var navigationBarImage: UIImageView!
+    private var navBarAppearance: UINavigationBarAppearance!
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        view.backgroundColor = .white
+        
+        let appearance = UINavigationBarAppearance()
+        appearance.configureWithDefaultBackground()
+        appearance.backgroundColor = .blue
+        
+        navigationController?.navigationBar.standardAppearance = appearance
+            navigationController?.navigationBar.scrollEdgeAppearance = appearance
+            navigationController?.navigationBar.compactAppearance = appearance
+        
+        buildViews()
+        styleViews()
+        constraintViews()
+    }
+    
+    private func buildViews(){
+        searchBarView = SearchBarView(controller: self)
+        view.addSubview(searchBarView)
+        
+        homeScreenMoviesView = HomeScreenMoviesView()
+        view.addSubview(homeScreenMoviesView)
+        
+        focusedHomeScreenView = HomeScreenFocusedView()
+        view.addSubview(focusedHomeScreenView)
+        
+        navigationBarAppName = UIView()
+        navigationBarImage = UIImageView(image: UIImage(named: "NavBarAppLogo"))
+        navigationBarAppName.addSubview(navigationBarImage)
+        
+        navBarAppearance = UINavigationBarAppearance()
+        
+    }
+    
+    private func styleViews(){
+        navBarAppearance.configureWithDefaultBackground()
+        navBarAppearance.backgroundColor = UIColor(red: 0.043, green: 0.145, blue: 0.247, alpha: 1)
+        navigationItem.titleView = navigationBarAppName
+        
+        navigationController?.navigationBar.standardAppearance = navBarAppearance
+        navigationController?.navigationBar.scrollEdgeAppearance = navBarAppearance
+        navigationController?.navigationBar.compactAppearance = navBarAppearance
+    }
+    
+    private func constraintViews(){
+        searchBarView.snp.makeConstraints(){
+            $0.top.equalTo(view.safeAreaLayoutGuide).offset(20)
+            $0.leading.trailing.equalToSuperview()
+            $0.height.equalTo(45)
+        }
+        
+        homeScreenMoviesView.snp.makeConstraints(){
+            $0.top.equalTo(searchBarView.snp.bottom).offset(10)
+            $0.leading.trailing.bottom.equalToSuperview()
+        }
+        
+        navigationBarImage.snp.makeConstraints(){
+            $0.centerX.equalToSuperview()
+            $0.centerY.equalToSuperview()
+            $0.height.equalTo(25)
+            $0.width.equalTo(170)
+        }
+    }
+    
+    public func switchViews(condition: Int){
```

Comment:
This is a really bad way to handle this. The condition being Int makes it really difficult for the caller of the function to know what they need to send as an argument.
If you changed the definition of your function to something like: 

`public func updateLayout(searchBarActive: Bool) { ... }`

it is automatically obvious that 

1. the function is changing the layout 
2. the layout depends on search bar being active or inactive 

The caller of the function won't be confused about what they need to send as an argument.